src/web/app/pages-instructor/instructor-session-individual-extension-page/instructor-session-individual-extension-page.component.ts

```diff
@@ -0,0 +1,527 @@
+    modalRef.componentInstance.feedbackSessionTimeZone = this.feedbackSessionTimeZone;
+
+    modalRef.componentInstance.onConfirmExtensionCallBack.subscribe((isNotifyDeadlines: boolean) => {
+      this.handleCreateAndDeleteDeadlines(selectedStudents, selectedInstructors, DeadlineHandlerType.CREATE,
+        isNotifyDeadlines, extensionTimestamp);
+      modalRef.componentInstance.isSubmitting = false;
+      modalRef.close();
+    });
+  }
+
+  /**
+   * Handles the opening the confirmation modal to delete deadlines.
+   */
+  onDelete(): void {
+    const modalRef: NgbModalRef = this.ngbModal.open(IndividualExtensionConfirmModalComponent);
+    const selectedStudents = this.getSelectedStudentsWithExtensions();
+    const selectedInstructors = this.getSelectedInstructorsWithExtensions();
+    modalRef.componentInstance.modalType = ExtensionModalType.DELETE;
+    modalRef.componentInstance.selectedStudents = selectedStudents;
+    modalRef.componentInstance.selectedInstructors = selectedInstructors;
+    modalRef.componentInstance.extensionTimestamp = this.feedbackSessionEndingTimestamp;
+    modalRef.componentInstance.feedbackSessionTimeZone = this.feedbackSessionTimeZone;
+
+    modalRef.componentInstance.onConfirmExtensionCallBack.subscribe((isNotifyDeadlines: boolean) => {
+      this.handleCreateAndDeleteDeadlines(selectedStudents, selectedInstructors, DeadlineHandlerType.DELETE,
+        isNotifyDeadlines);
+      modalRef.componentInstance.isSubmitting = false;
+      modalRef.close();
+    });
+  }
+
+  private handleCreateAndDeleteDeadlines(
+    selectedStudents: StudentExtensionTableColumnModel[],
+    selectedInstructors: InstructorExtensionTableColumnModel[],
+    updateDeadlinesType: DeadlineHandlerType,
+    isNotifyDeadlines: boolean,
+    extensionTimestamp?: number,
+  ): void {
+    const request: FeedbackSessionUpdateRequest = {
+      studentDeadlines: this.getUpdatedDeadlines(selectedStudents, this.studentDeadlines,
+        updateDeadlinesType, extensionTimestamp),
+      instructorDeadlines: this.getUpdatedDeadlines(selectedInstructors, this.instructorDeadlines,
+        updateDeadlinesType, extensionTimestamp),
+      ...this.feedbackSessionDetails,
+    };
+
+    this.isSubmittingDeadlines = true;
+    this.feedbackSessionsService
+      .updateFeedbackSession(this.courseId, this.feedbackSessionName, request, isNotifyDeadlines)
+      .pipe(finalize(() => { this.isSubmittingDeadlines = false; }))
+      .subscribe(() => {
+          this.loadFeedbackSessionAndIndividuals();
+          this.showSucessToast(updateDeadlinesType, selectedStudents, selectedInstructors);
+        },
+        (resp: ErrorMessageOutput) => {
+          this.statusMessageService.showErrorToast(resp.error.message);
+        },
+      );
+  }
+
+  private showSucessToast(updateDeadlinesType: DeadlineHandlerType,
+    selectedStudents: StudentExtensionTableColumnModel[],
+    selectedInstructors: InstructorExtensionTableColumnModel[]): void {
```

Comment:
Let's make the following change to match the other functions, else it looks like the parameters are part of the method body.

```suggestion
  private showSucessToast(updateDeadlinesType: DeadlineHandlerType,
    selectedStudents: StudentExtensionTableColumnModel[],
    selectedInstructors: InstructorExtensionTableColumnModel[]
  ): void {
```